UserSampleId/app/src/androidTest/java/com/swrve/usersampleid/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.swrve.usersampleid;
```

Comment:
Not sure on this one, it is auto created by Android studio. I'll check if it is necessary